src/capi/azext_capi/custom.py

```diff
@@ -152,7 +153,7 @@ def find_cluster_to_become_management_cluster():
     return cluster_name
 
 
-def create_management_cluster(cmd, yes=False):
+def create_management_cluster(cmd, yes=False, cluster_name=None):
```

Comment:
```suggestion
def create_management_cluster(cmd, cluster_name=None, yes=False):
```

The `yes` argument is usually the last one in these az CLI functions (at least that's the convention I've seen). It shouldn't change the code's behavior here either way, so we should probably do the conventional thing.

Reply:
Also, could you add some help text in `_help.py` for the new `--cluster-name` argument?

```
% az capi management create -h
This command is from the following extension: capi

Command
    az capi management create : Create a CAPI management cluster.
        Command group 'capi management' is in preview and under development. Reference and
        support levels: https://aka.ms/CLI_refstatus
Arguments
    --cluster-name
    --yes -y           : Do not prompt for confirmation.
```